railties/test/generators/plugin_generator_test.rb

```diff
@@ -508,6 +508,43 @@ def test_generating_controller_inside_mountable_engine
       assert_match(/namespace :admin/, contents)
       assert_no_match(/namespace :bukkit/, contents)
     end
+    assert_file "test/controllers/bukkits/admin/dashboard_controller_test.rb" do |contents|
+      assert_match(/@routes = Engine.routes/, contents)
+    end
+  end
+
+  def test_controller_tests_pass_by_default_inside_mountable_engine
+    run_generator [destination_root, "--mountable"]
+    FileUtils.cd(destination_root)
+
+    quietly { `bin/rails g controller dashboard foo` }
+
+    assert_match(/2 runs, 2 assertions, 0 failures, 0 errors/, `bundle exec rake test 2>&1`)
+  end
+
+  def test_generating_scaffold_controller_inside_mountable_engine
```

Comment:
do we still need this test? Looks like a duplication of the one following right after.